source/wp-content/themes/wporg-news-2021/blocks/post-type-name/index.php

```diff
@@ -0,0 +1,89 @@
+<?php
```

Comment:
🤔  It might be better to have this in https://github.com/WordPress/wporg-mu-plugins/, since it's not really specific to this theme.

That might be overkill, though. I don't feel strongly.

Reply:
I think I'll leave it here for now. Should we need it, move it later.

Reply:
👍🏻